packages/react-components/src/components/chart/eChartsConstants.ts

```diff
@@ -63,3 +60,10 @@ export const DEFAULT_ECHART_OPTIONS: EChartsOption = {
   tooltip: DEFAULT_TOOLTIP,
   dataZoom: DEFAULT_DATA_ZOOM,
 };
+
+// tool tip constants
+export const tooltipColors = ['#DA7596', '#2EA597', '#688AE8'];
```

Comment:
Can we just use the colors we use for trendlines? I think theyll need to match right?

Reply:
ah! this is a leftover, don't think we need this.